Add label selector (#32129)
home-assistant/home-assistant.github.io: source/_docs/blueprint/selectors.markdown

The output of this selector is a string containing the selected icon,
for example: `mdi:bell`.

## Label selector

> The label selector shows a label finder the can pick labels. The value of the
> input will be the label ID. If `multiple` is set to `true`, the value is a list
> of label IDs.

Reviewer: ```suggestion
The label selector shows a label finder that can pick labels. The value of the
input is the label ID. If `multiple` is set to `true`, the value is a list
of label IDs.
```
